frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -696,18 +696,16 @@ export class HtmlDocument extends React.Component {
   shouldDisplayNotifyAllMessage = () => {
     const { state } = this
 
+    const lastModifierIsLoggedUser = (content) => {
+      return content.last_modifier && content.last_modifier.user_id === state.loggedUser.userId
```

Comment:
```suggestion
      return content.last_modifier && content.last_modifier.user_id === this.state.loggedUser.userId
```

Reply:
Do you think it doesn't work? As the function is a closure the `state` object should be accessible to it, right?